src/custom_nodes/model_zoo_intel_object_detection/README.md

```diff
@@ -30,6 +30,11 @@ make
 ```
 It will compile the library inside a docker container and save the results in `lib` folder.
 
+You can also select base OS between RH 8.4 (redhat), CentOS 7 (centos) and Ubuntu 20.04 (ubuntu) by using `BASE_OS` environment variable.
+```
+BASE_OS=redhat make
```

Comment:
Shouldn't make be in a new line?

Reply:
No, we execute make command with enviroment variable `BASE_OS` set to redhat. If we separate the lines we will not get the enviroment variable passed to the context of make.